src/otx/algo/segmentation/backbones/litehrnet.py

```diff
@@ -1500,10 +1290,11 @@ class LiteHRNetBackbone:
         },
     }
 
-    def __new__(cls, version: str) -> NNLiteHRNet:
+    def __new__(cls, model_name: str) -> LiteHRNetModule:
         """Constructor for LiteHRNet backbone."""
-        if version not in cls.LITEHRNET_CFG:
-            msg = f"model type '{version}' is not supported"
+        if model_name not in cls.LITEHRNET_CFG:
+            msg = f"model type '{model_name}' is not supported"
             raise KeyError(msg)
-
-        return NNLiteHRNet(**cls.LITEHRNET_CFG[version])
+        stem_configuration = cls.LITEHRNET_CFG[model_name].pop("stem", {})
```

Comment:
Just a suggestion, how about using a common name such as `config` or `default_config` across all tasks?

Reply:
I left the same comment in https://github.com/openvinotoolkit/training_extensions/pull/3864#discussion_r1724377555

Reply:
I expect this can be resolved in one PR after discussion.